Podfile

```diff
@@ -1,17 +1,26 @@
+project_name='ProjectTemplate'
+
 platform :ios, '10.3'
-project 'ProjectTemplate', 'Beta-Development' => :release, 'Beta-Stage' => :release, 'Beta-Production' => :release, 'Release' => :release, 'Development' => :debug
+project project_name, 'Beta-Development' => :release, 'Beta-Stage' => :release, 'Beta-Production' => :release, 'Release' => :release, 'Development' => :debug
 
 inhibit_all_warnings!
 use_frameworks!
 
-target 'ProjectTemplate' do
+workspace project_name + '.xcworkspace'
+
+target project_name do
+    project project_name + '.xcodeproj'
     pod 'SwiftLint', '~> 0.27'
     
-    pod 'ACKLocalization', '~> 0.3'
     pod 'SwiftGen', '~> 6.0'
     pod 'Smartling.i18n', '~> 1.0'
     
-    target 'UnitTests' do
+    target 'ProjectTemplateTests' do
```

Comment:
Is it really correct that the target here is named "ProjectTemplateTests"?

Reply:
This was done in an effort to be as easy as possible to add another project.
Adding a new project means using `project` definition from `Project+Templates` where the `tests` target is tied to the new project - which would mean a duplicated target if we change that to `UnitTests` and do not take the project's name into account.